sdk/core/Azure.Core.TestFramework/src/ResourceCleanup.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.ResourceManager.Resources;
+using System.Collections.Generic;
+
+namespace Azure.Core.TestFramework
+{
+    public static class ResourceCleanup
+    {
+
+        public static void cleanUpResources(IEnumerable<string> resourceGroups, string subscriptionId, TokenCredential credential)
```

Comment:
Does this method have to be in a separate file, I only see a single usage from `ManagementRecordTestBase`.

Reply:
We wanted a generic cleanup so that when we start using ARM template for setup at some time in the future (multiple resources groups being made by arm template will be possible). The class that handle the resource group creation via arm template will call this to teardown the resource groups the ARM template made.

Reply:
I thought we are running the ARM templates before invoking the test. Are we planning to have ARM template deployed during the test runtime?

Reply:
The plan is to support both. An ARM template can be executed via OneTimeSetup and the names of the resources created read into the test. Also, a user could manually run an ARM template as they do now.